index.html

```diff
@@ -9,37 +9,213 @@
+                <input type="button" value="bootstrap" class="tech-item item-three"/>
+                <input type="button" value="Ruby" class="tech-item item-four"/>
+            </div>
+            <button type="button" id="post-btn">See Project</button>
+        </div>
+        <div class="job-item-two">
+        <h1 class="job-post-title">Profesional Art Printing Data</h1>
+        <div class="job-post">
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        </div>
+        <div class="job-tech-lang">
+            <input type="button" value="html" class="job-item job-one"/>
+            <input type="button" value="bootstrap" class="job-item job-one"/>
+            <input type="button" value="Ruby" class="job-item job-one"/>
+        </div>
+        </div>
+        <button type="button" class="see-jobs-btn">See Project</button>
+        <div class="job-item-two">
+            <h1 class="job-post-title">Profesional Art Printing Data</h1>
+            <div class="job-post">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            </div>
+            <div class="job-tech-lang">
+            <input type="button" value="html" class="job-item job-one"/>
+            <input type="button" value="bootstrap" class="job-item job-one"/>
+            <input type="button" value="Ruby" class="job-item job-one"/>
+            </div>
+        </div>
+        <button type="button" class="see-jobs-btn">See Project</button>
+        <div class="job-item-two">
+            <h1 class="job-post-title">Profesional Art Printing Data</h1>
+            <div class="job-post">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            </div>
+            <div class="job-tech-lang">
+            <input type="button" value="html" class="job-item job-one"/>
+            <input type="button" value="bootstrap" class="job-item job-one"/>
+            <input type="button" value="Ruby" class="job-item job-one"/>
+            </div>
+        </div>
+        <button type="button" class="see-jobs-btn">See Project</button>
+        <div class="job-item-two">
+            <h1 class="job-post-title">Profesional Art Printing Data</h1>
+            <div class="job-post">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+            </div>
+            <div class="job-tech-lang">
+            <input type="button" value="html" class="job-item job-one"/>
+            <input type="button" value="bootstrap" class="job-item job-one"/>
+            <input type="button" value="Ruby" class="job-item job-one"/>
+            </div>
+        </div>
+        <button type="button" class="see-jobs-btn">See Project</button>
```

Comment:
- [ ] I can see that for each project you don't have its own container, this will make things complicated when you need to work on the desktop version, so kindly let's have a `<div>` tag to wrap the content of each project 👍 
- [ ] Kindly let's add button interactions on all the buttons, like hover, active... as in the project requirement 👇 
![image](https://user-images.githubusercontent.com/55465133/133862664-a3aa77a6-38bf-44ad-98f3-477a4e350afd.png)
![image](https://user-images.githubusercontent.com/55465133/133862676-e602fa8d-dd9b-48ef-a349-7fd9084224b9.png)